src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
 body {
+  width: 100%;
+  height: 100%;
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  position: absolute;
+  inset: 0;
+  margin: auto;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minute-hand {
+    margin: auto;
+    position: absolute;
+    left: 38.5vmin;
+    top: 20vmin;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    z-index: -1;
+
+    @include animation-minute-hand;
+
+    animation-duration: 3600s;
+
+    &--speed-up {
+      animation-duration: 600s;
+    }
+  }
```

Comment:
It's a good practice to add a brief comment describing the purpose or functionality of each block of styles. This will make your code more readable and maintainable for others (and for yourself in the future). For example, you could add a comment here explaining what the `__minute-hand` class is for.